tests/testthat/test-parser.R

```diff
@@ -0,0 +1,44 @@
+test_that("Using Perceval version 0.12.24", {
+  tools_path <- "../../tools.yml"
+  tool <- yaml::read_yaml(tools_path)
+  perceval_path <- tool[["perceval"]]
```

Comment:
`perceval_path <- path.expand(perceval_path)`
Needs to be added after this line, or it will cause the test to fail (~/ paths will not resolve without this).